scrunch/datasets.py

```diff
@@ -3410,7 +3410,8 @@ def create_tmp_ds(self, csv_file):
             "table": {
                 "element": "crunch:table",
                 "metadata": metadata
-            }
+            },
+            "project": self.dataset.project.url,
```

Comment:
@jjdelc you think it's okay to use the same project as the source dataset?

Reply:
Yes absolutely, because it ensures you have write permissions.